backstage/templates/scaffolder/add-repo/content/.markdownlint-cli2.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
this config file doesn't make sense if we're not calling the markdownlint CI. If we're going to be enabling that as a standard, I'd like to discuss these settings; I think both of the rules getting disabled here are Quite Good Actually

Reply:
hmm good point. The file itself is just copy pasta, I'm happy to figure out what defaults we want, though maybe in another PR, that way we can change all the templates? 

Do you think I should remove this file, or add workflow?

Reply:
remove -- I never copied this file in any of my docs repos, so I don't think it's really codifying an existing standard.